src/openforms/forms/templates/admin/forms/formsubmissionstatistics/search_form.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n static %}
```

Comment:
What is the purpose of overriding this?

Reply:
Removing the item result count, as it shows the wrong number. See commit message https://github.com/open-formulieren/open-forms/pull/5015/commits/719dbb4697bcc38ace68c8cca3f1e717d006fcf3 :)